p2p/sentry/sentry_grpc_server.go

```diff
@@ -903,20 +902,6 @@ func (ss *GrpcServer) SendMessageByMinBlock(_ context.Context, inreq *proto_sent
 
 func (ss *GrpcServer) SendMessageById(_ context.Context, inreq *proto_sentry.SendMessageByIdRequest) (*proto_sentry.SentPeers, error) {
 	reply := &proto_sentry.SentPeers{}
-	msgcode := eth.FromProto[ss.Protocols[0].Version][inreq.Data.Id]
-	if msgcode != eth.GetBlockHeadersMsg &&
-		msgcode != eth.BlockHeadersMsg &&
-		msgcode != eth.GetBlockBodiesMsg &&
-		msgcode != eth.BlockBodiesMsg &&
-		msgcode != eth.NewBlockMsg &&
-		msgcode != eth.NewBlockHashesMsg &&
-		msgcode != eth.GetReceiptsMsg &&
-		msgcode != eth.ReceiptsMsg &&
-		msgcode != eth.NewPooledTransactionHashesMsg &&
-		msgcode != eth.PooledTransactionsMsg &&
-		msgcode != eth.GetPooledTransactionsMsg {
```

Comment:
Why do these checks need to be removed?

Reply:
Because in this function all possible message codes were listed, so I take it that the intention was to simply check that the message passed is of a known type rather than to filter out some message types.